src/index.js

```diff
@@ -58,6 +65,12 @@ module.exports = options => ({
   // Fix sourcemaps
   result(stylelintResult, filepath) {
     const err = errorWasThrown[filepath]
+    const mappedOptions = Object.assign({}, DEFAULT_OPTIONS, options)
+    const absolutePath = path.resolve(process.cwd(), filepath)
```

Comment:
Maybe `filepath` here is the absolute path already, or those `taggedTemplateLocs/interpolationLines` can't get value in old codes.

Reply:
Compared them. Were the same with the files I tested.
Therefore removed the `absolutePath`